app.py

```diff
@@ -1,14 +1,15 @@
-from flask import Flask, render_template
+from flask import Flask, render_template, request
 from dice import roll_die
 
 app = Flask(__name__)
 
 
-@app.route("/")
-@app.route("/dice_game")
+@app.route('/')
+@app.route('/dice_game')

```

Comment:
I think you can remove this line.

Reply:
Done. Have kept the following line ... @app.route('/dice_game')

Reply:
Typically, urls don't use underscores. 
```suggestion
@app.route('/dice-game')

```